crates/oci-runtime-clap/README.md

```diff
@@ -0,0 +1,27 @@
+# oci-runtime-clap
+
+This is a crate to parse command line arguments for OCI container
+runtimes as specified in the [OCI Runtime Command Line
+Interface][https://github.com/opencontainers/runtime-tools/blob/master/docs/command-line-interface.md).
+
+## Implemented subcommands
+
+| Command    | oci-runtime-clap | CLI Specification | runc | crun | youki |
+| :--------: | :--------------: | :---------------: | :--: | :--: | :---: |
+| create     | ✅               | ✅                | ✅   | ✅   | ✅    |
+| start      | ✅               | ✅                | ✅   | ✅   | ✅    |
+| state      | ✅               | ✅                | ✅   | ✅   | ✅    |
+| kill       | ✅               | ✅                | ✅   | ✅   | ✅    |
+| delete     | ✅               | ✅                | ✅   | ✅   | ✅    |
+| :--------: | :--------------: | :---------------: | :--: | :--: | :---: |
+| checkpoint |                  |                   | ✅   | ✅   |       |
+| events     | ✅               |                   | ✅   |      | ✅    |
+| exec       | ✅               |                   | ✅   | ✅   | ✅    |
+| list       | ✅               |                   | ✅   | ✅   | ✅    |
+| pause      | ✅               |                   | ✅   | ✅   | ✅    |
+| ps         | ✅               |                   | ✅   | ✅   | ✅    |
+| restore    |                  |                   | ✅   | ✅   |       |
+| resume     | ✅               |                   | ✅   | ✅   | ✅    |
+| run        | ✅               |                   | ✅   | ✅   | ✅    |
+| spec       | ✅               |                   | ✅   | ✅   | ✅    |
+| update     |                  |                   | ✅   | ✅   |       |
```

Comment:
'info' is missing 😉

Reply:
Since `info` is an extended command, perhaps it's not written on purpose? Even if that's the case, I'd like to write somewhere that there is an info command.

Reply:
`info` is not implement in *this* crate, so it doesn't belong in this list.  `info` is already described in several places in the `youki` crate - most notably in the output of `youki --help`.

Reply:
Ah got it